src/Node/Express/Middleware/Static.js

```diff
@@ -0,0 +1,11 @@
+// module Node.Express.Middleware.Static
+"use strict";
+
+exports._static = function(root) {
+  return function(req, res, nxt) {
+    return function() {
+      var app = require("express");
```

Comment:
I'd prefer to name it `express` instead of `app` because `app` has a notion of application being developed and served.